confero/settings.py

```diff
@@ -76,16 +79,28 @@
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.getenv("DB_NAME", "confero"),
-        'USER': os.getenv("DB_USER", "postgres"),
-        'PASSWORD': os.getenv("DB_PASSWORD", "postgres"),
-        'HOST': os.getenv("DB_HOST", "localhost"),
-        'PORT': os.getenv("DB_PORT", "5432"),
+if 'RDS_DB_NAME' in os.environ:
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql',
+            'NAME': os.environ['RDS_DB_NAME'],
```

Comment:
It'd be nice to not have to duplicate all these config lines. What do you think about something like:

```python
db_prefix = 'RDS_DB_'  if 'RDS_DB_NAME' in os.environ else 'DB_'

'default': {
            'ENGINE': 'django.db.backends.postgresql',
            'NAME': os.environ[db_prefix + 'NAME'],
            ...
```

Reply:
smart. I like it a lot.